extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -22,14 +22,20 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 		var id;
 
 		log(['dispatchEvent', event], 'info', logGroup);
-
 		for (id in registeredCallbacks) {
-			if (registeredCallbacks.hasOwnProperty(id) &&
-				registeredCallbacks[id] && event.slot && event.slot === googleSlots.getSlot(id)
-			) {
-				log(['dispatchEvent', event, id, 'Launching registered callback'], 'debug', logGroup);
-				registeredCallbacks[id](event);
-				return;
+			if (registeredCallbacks.hasOwnProperty(id)) {
+				var slotId = id;
```

Comment:
You need `slotId` only for `googleSlots.getSlot(slotId)`, maybe close this abstraction behind some additional function